src/world_renderer.rs

```diff
@@ -182,7 +193,8 @@ impl WorldRenderer {
                         // We use OpenGL's instancing feature which allows us to render huge amounts of
                         // cubes at once.
                         // OpenGL instancing = instead of setting 1000 times different uniforms, you give once 1000 attributes
-                        let position_buffer = glium::VertexBuffer::dynamic(&display, &self.world.get_cubes_to_draw(self.cam.touched_cube())).unwrap();
+                        self.world.set_selected_cube(self.cam.touched_cube());
+                        let position_buffer = glium::VertexBuffer::immutable(&display, self.world.cube_to_draw()).unwrap();
```

Comment:
I am assuming that there is a reason why you changed to `VertexBuffer::immutable`. I just put this comment so that you can explain me why :)

Reply:
Once created VertexBuffer is not modified so dynamic is not needed and I saw a small speed up of fps using immutable